changelog.d/12772.bugfix

```diff
@@ -0,0 +1 @@
+Moves methods that call `add_push_rule` to the `PushRuleStore` class.
```

Comment:
Since (fortunately) this doesn't impact users at present, can you rename this to `12772.misc` please?

Additionally can this say "Move" rather than "Moves" please, to fit in with the style of our other changelogs?